cmake/Modules/FindGoogleSanitizer.cmake

```diff
@@ -0,0 +1,97 @@
+# Find google sanitizers
```

Comment:
Hmm, that's the first time I hear to them referred by that name. Can you please share some background about origin of the name and the file?

Reply:
Oh they are proposed by google. And the GitHub repository https://github.com/google/sanitizers. So I used the name.

Reply:
Sure, I understand the historic aspect, but I doubt that implementation in gcc or VS2022 is actually done by google,  so I wonder why not rename it to something vendor-agnostic, say `FindSanitizers.cmake`

@ezyang, what do you think?

Reply:
I don't really care, PR submitter's privilege :P

Reply:
Adhere to the advice of @malfet